lib/ui/lobby.dart

```diff
@@ -94,33 +96,12 @@ class _LobbyScreenState extends State<LobbyScreen> {
                           Expanded(
                             flex: 15,
                             child: SizedBox(
-                              height: appBarHeight * 0.75,
+                              height: appBarHeight * 0.8,
                               width: screenWidth * 0.55,
-                              child: TextField(
-                                controller: myController,
-                                keyboardType: TextInputType.number,
-                                inputFormatters: <TextInputFormatter>[
-                                  FilteringTextInputFormatter.digitsOnly
-                                ],
```

Comment:
Can you keep the TextField keyboard with input type number and allowing digitsOnly?

The room ID consists of only numerical digits, and opening the numerical keyboard and allowing only numbers facilitates input.

Reply:
added keyboardType propriety

Now AppTextField can  change keyboard
updated lobby ApptextField keyboard to numerical keyboard